examples/Serving-An-XGboost-Model-With-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>You renamed the objects in the code to shorten them. We need to update the names in the text.</p><p> </p><p>Triton Inference Server -&gt; the first letter of each word is upper case</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Good point -- changes made